scripts/oss-fuzz.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1+
+#
+# Copyright 2017 Jonathan Rudenberg
+#
+# systemd is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# systemd is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with systemd; If not, see <http://www.gnu.org/licenses/>.
+
+set -ex
+
+export LC_CTYPE=C.UTF-8
+
+meson $WORK -Doss-fuzz=true -Db_lundef=false
```

Comment:
Would it be better to rebuild everything from scratch, so that `python infra/helper.py build_fuzzers systemd` won't fail when it is run for the second time:
```sh
$ sudo python infra/helper.py build_fuzzers systemd
...
+ ninja -C /work fuzzers
ninja: Entering directory `/work'
[3/4] Running external command fuzzers.
+ df=/work/dns-fuzzing
+ rm -rf /work/dns-fuzzing
+ git clone --depth 1 https://github.com/CZ-NIC/dns-fuzzing /work/dns-fuzzing
Cloning into '/work/dns-fuzzing'...
remote: Counting objects: 9329, done.
remote: Compressing objects: 100% (6050/6050), done.
remote: Total 9329 (delta 138), reused 9328 (delta 138), pack-reused 0
Receiving objects: 100% (9329/9329), 1.39 MiB | 614.00 KiB/s, done.
Resolving deltas: 100% (138/138), done.
Checking connectivity... done.
+ zip -jqr /out/fuzz-dns-packet_seed_corpus.zip /work/dns-fuzzing/packet
+ mkdir -p /out/src/shared
+ mv /work/src/shared/libsystemd-shared-236.so /out/src/shared
+ find /work -maxdepth 1 -type f -executable -name 'fuzz-*' -exec mv '{}' /out ';'
+ mv '/work/*.so' src/fuzz/fuzz-dns-packet.options /out
mv: cannot stat '/work/*.so': No such file or directory
fuzzers build failed.
```
?

Reply:
Makes sense, done.